src/python/pants/backend/python/subsystems/debugpy.py

```diff
@@ -42,13 +58,13 @@ def _get_main_spec_args(main: MainSpecification) -> tuple[str, ...]:
             return ("-m", main.module)
 
         assert isinstance(main, ConsoleScript)
-        # NB: This is only necessary while debugpy supports Py 3.7, as `importlib.metadata` was
-        # added in Py 3.8 and would allow us to resolve the console_script using just the stdlib.
         return (
             "-c",
             (
-                "from pex.third_party.pkg_resources import working_set;"
-                + f"(next(working_set.iter_entry_points('console_scripts', '{main.name}')).resolve())()"
+                "import importlib_metadata;"
+                + "eps = importlib_metadata.entry_points()['console_scripts'];"
+                + f"ep = next(ep for ep in eps if ep.name == '{main.name}');"
+                + "ep.load()()"
```

Comment:
Softwrap?

Reply:
Software, to me, is for uaer-visible strings.

We could do ";".join(...), But also meh.

I just don't like implicit string concat